src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -40,6 +40,14 @@ object EventModule {
   final val busName = "events"
 }
 
+/** Local leadership events. They are not delivered via the event endpoints. */
+sealed trait LeadershipStateEvent
```

Comment:
I wonder, if we should make the concept of "Local Events" more visible (like exception hierarchy for checked, runtime exceptions).
See my comments below, where the question is: do we need "Local Events" for this.